.circleci/common.sh

```diff
@@ -14,6 +14,8 @@ fi
 # 2. CONDA_PREFIX (if it exists)
 # 3. The conda install directory (if it exists)
 export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH:-${CONDA_PREFIX:-"$(dirname $(which conda))/../"}}
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$(python3-config --prefix)/lib"
```

Comment:
How hard would it be to convert this constellation of testing bash scripts to bazel [`test_suite`s](https://bazel.build/reference/be/general#test_suite)? Could bazel handle setting env vars, flags, etc? Would we need custom logic to generate targets for different variants (e.g. TPU, single-GPU, multi-GPU, etc)?

This PR is already quite large, so no need to make that change now